readme.md

```diff
@@ -426,6 +426,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Bioinformatics](https://github.com/danielecook/Awesome-Bioinformatics)
 - [Colorful](https://github.com/Siddharth11/Colorful) - Resources to choose your next color scheme.
 - [Steam](https://github.com/scholtzm/awesome-steam) - Digital distribution platform.
+- [Bots](https://github.com/hackerkid/bots) - Resources on building bots.
```

Comment:
Don't like everything to be prefixed with "resources". All lists contain resources. Maybe just `Building bots` or something.